### ISSUE TITLE
Discussing the sequence-wise batch normalization

### ISSUE THREAD

**Issue by AzizCode92** [ScartleRoy/TF_LSTM_seq_bn#1]
First, thank you for your code.
As far as I understood, sequence-wise batch normalization is applying batch-norm on both the [batch] and [time] axis of your input. Please correct me if I am wrong.
The input to my lstm_cell is with the format [batch_size, input_size].
Following the code, the axis parameter would be [0].
Passing it to the tf.nn.moments inside the batch_norm function will make batch_norm only to batch axis and ignore the time axis.
Would it ok instead to set axis = [0,1] ? So that we can apply the mean and variance of both batch and time axis?
